OpenRA.Mods.Common/Traits/Render/RenderDetectionCircle.cs

```diff
@@ -29,8 +29,14 @@ class RenderDetectionCircleInfo : TraitInfo, Requires<DetectCloakedInfo>
 		[Desc("Color of the circle and scanner update line.")]
 		public readonly Color Color = Color.FromArgb(128, Color.LimeGreen);
 
+		[Desc("Range circle line width")]
+		public readonly float Width = 1;
+
 		[Desc("Contrast color of the circle and scanner update line.")]
-		public readonly Color ContrastColor = Color.FromArgb(96, Color.Black);
+		public readonly Color BorderColor = Color.FromArgb(96, Color.Black);
```

Comment:
I'm sorry to say that we'll need a (thankfully trivial) update rule for this.

Reply:
I kind of expected this to happen.